src/AppInstallerCLICore/Argument.cpp

```diff
@@ -149,6 +149,8 @@ namespace AppInstaller::CLI
             return { type, "all"_liv, 'r', "recurse"_liv, ArgTypeCategory::MultiplePackages };
         case Execution::Args::Type::IncludeUnknown:
             return { type, "include-unknown"_liv, 'u', "unknown"_liv };
+        case Execution::Args::Type::IncludePinned:
+            return { type, "include-pinned"_liv, "pinned"_liv, ArgTypeCategory::CopyFlagToSubContext };
         case Execution::Args::Type::UninstallPrevious:
             return { type, "uninstall-previous"_liv, ArgTypeCategory::InstallerBehavior };

```

Comment:
>uninstall-previous [](http://example.com/codeflow?start=28&length=18)

I guess this should be copied to subcontext as well since we support multiple installs using subcontexts?